Casks/macloggerdx.rb

```diff
@@ -9,7 +9,7 @@
 
   livecheck do
     url :homepage
-    regex(/Download:\s*v?(\d+(?:\.\d+)+)[^a-z]+/i)
+    regex(/Download:\s*v?(\d+(?:\.\d++)++)[^b]+/i)
```

Comment:
```suggestion
    regex(/Download:\s*v?(\d+(?:\.\d+)+)[\s<]+?/im)
```

In my view, both `[^a-z]+` and `[^b]+` feel a bit too loose in this context. The goal is to only match a version like `6.36` while omitting unstable versions like `6.37b14`, so I think it may make more sense to use `[\s<]+?` to anchor the end and restrict matching to versions like `6.36`.

Since this is matching loosely formatted text that can potentially span more than one line, this suggestion also uses the `m` flag.

Reply:
The change to using possessive quantifiers wasn't done for performance, but like you guessed was to avoid the unstable/beta version strings like '6.37b14' from matching the original regex mistakenly as '6.3'. 

> The goal is to only match a version like 6.36 while omitting unstable versions like 6.37b14, so I think it may make more sense to use [\s<] to anchor the end.

Agree, this approach is better, will make this change.